CONTRIBUTING.md

```diff
@@ -121,12 +130,12 @@ Install our test dependencies:
   $ conda install --file requirements/test_requirements.txt -c spyder-ide
 ```
 
-or using pip
+or using `pip` (not recommended):
```

Comment:
I think we should change `not recommended` to `for experts only`

Reply:
Done